### DISCUSSION TITLE
How can I increase width of toc-drawer?

### DISCUSSION THREAD

**Discussion by janluke** [pradyunsg/furo#166] [Questions] [answered]
I'm using a larger font-size for the side ToC, so I'd like to increase the width of it. Unfortunately, increasing the width breaks the responsive layout. Is there a workaround or a plan to make it larger by default?

Thank you.

**Comment by pradyunsg** [comment, marked as answer]
> Is there a workaround or a plan to make it larger by default?

No there isn't.

Using font size smaller than the left sidebar and content is an intentional aesthetic choice (prevents that from grabbing attention, and make it to look more like an aside).

And, making it possible to tweak the width of toc-drawer would also imply making it possible to tweak the content width, which IMO is basically a strong NOPE.
